README.md

```diff
@@ -184,3 +187,9 @@ This project is licensed under the MIT License. See the [LICENSE.md](https://git
 ## Acknowledgements
 
 The "Kimera" name has been inspired from greek mythology - [Chimera](<https://en.wikipedia.org/wiki/Chimera_(mythology)>)
```

Comment:
, and the logo harkens back to the graphql logo. (wanted to add a little more as the sentence just ends rather abruptly. but no need to add this in if you think otherwise 😄  ).